renderdoc/driver/vulkan/vk_counters.cpp

```diff
@@ -134,7 +134,15 @@ void VulkanReplay::convertKhrCounterResult(CounterResult &rdcResult,
   if(khrUnit == VK_PERFORMANCE_COUNTER_UNIT_NANOSECONDS_KHR)
   {
     RDCASSERT(type == CompType::Double);
-    rdcResult.value.d /= 1000.0 * 1000.0 * 1000.0;
+
+    if(khrStorage == VK_PERFORMANCE_COUNTER_STORAGE_FLOAT64_KHR)
```

Comment:
This should also check for `FLOAT32` here as that will be stored in the `d` member as well.